aws-apprunner/01-variables.tf

```diff
@@ -153,6 +153,18 @@ variable "instance_memory" {
   default     = 2048
 }
 
+variable "pvault_devmode" {
+  description = "Enable devmode for Pvault. See https://piiano.com/docs/guides/configure/environment-variables#production-and-development-mode for more details."
+  type        = bool
+  default     = false
+}
+
+variable "pvault_env_vars" {
```

Comment:
IS it good practice that part of the env var will be explicitly and part of them will aggregated in map ?